core/src/IdToken.cs

```diff
@@ -72,6 +74,12 @@ internal class IdToken
         [DataMember(Name = IdTokenClaim.HomeObjectId, IsRequired = false)]
         public string HomeObjectId { get; set; }
 
+        [DataMember(Name = IdTokenClaim.GivenName)]
```

Comment:
this one doesn't need IsRequired = false?

Reply:
it is optional, which means that other sdk might not persist it

Reply:
I see. thanks.

---
In reply to: [227982217](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1350#discussion_r227982217) [](ancestors = 227982217)